spec/lib/rex/proto/kerberos/credential_cache/krb5_ccache_presenter_spec.rb

```diff
@@ -0,0 +1,255 @@
+                    Logoff Time: Never Expires (inf)
+                    Kick Off Time: Never Expires (inf)
+                    Password Last Set: No Time Set (0)
+                    Password Can Change: No Time Set (0)
+                    Password Must Change: Never Expires (inf)
+                    Logon Count: 0
+                    Bad Password Count: 0
+                    User ID: 500
+                    Primary Group ID: 513
+                    User Flags: 0
+                    User Session Key: \\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00
+                    User Account Control: 528
+                    Sub Auth Status: 0
+                    Last Successful Interactive Logon: No Time Set (0)
+                    Last Failed Interactive Logon: No Time Set (0)
+                    Failed Interactive Logon Count: 0
+                    SID Count: 0
+                    Resource Group Count: 0
+                    Group Count: 5
+                    Group IDs:
+                      Relative ID: 513, Attributes: 7
+                      Relative ID: 512, Attributes: 7
+                      Relative ID: 520, Attributes: 7
+                      Relative ID: 518, Attributes: 7
+                      Relative ID: 519, Attributes: 7
+                    Logon Domain ID: S-1-5-21-3541430928-2051711210-1391384369
+                    Effective Name: 'Administrator'
+                    Full Name: ''
+                    Logon Script: ''
+                    Profile Path: ''
+                    Home Directory: ''
+                    Home Directory Drive: ''
+                    Logon Server: ''
+                    Logon Domain Name: 'WINDOMAIN.LOCAL'
+                  Client Info:
+                    Name: 'Administrator'
+                    Client ID: 2022-11-28 15:51:29 +0000
+                  Pac Server Checksum:
+                    Signature: \\x5e\\xb9\\x40\\x0b\\xca\\xb4\\x2b\\xab\\xcd\\x59\\x82\\x10
+                  Pac Private Server Checksum:
+                    Signature: \\x58\\x58\\xaf\\xf1\\x9f\\x89\\xc9\\xb9\\xce\\x01\\xc4\\x37
+    EOF
+  end
+
+  describe '#present' do
+    context 'when decryption key is not provided' do
+      it 'returns a string not containing pac info' do
+        expect(subject.present).to eq(no_decryption_example)
+      end
+    end
+
+    context 'when the incorrect decryption key is provided' do
+      it 'raises an exception' do
+        expect { subject.present(key: "a wrong key that's 32 bytes long") }.to raise_error(Rex::Proto::Kerberos::Model::Error::KerberosError)
+      end
+    end
+
+    context 'when the decryption key is invalid' do
+      it 'raises an exception' do
+        expect { subject.present(key: 'Invalid key') }.to raise_error(ArgumentError)
+      end
+    end
+
+    context 'when the correct decryption key is provided' do
+      it 'returns a string containing pac info' do
+        puts subject.present(key: key)
```

Comment:
```suggestion
```

Reply:
I have also removed this but it's still showing for some reason